widget/playground/src/constants/fonts.ts

```diff
@@ -2,13 +2,82 @@ export const FONTS = [
   {
     name: 'Roboto',
     value: 'Roboto',
+    url: '',
   },
   {
     name: 'Times New Roman',
     value: 'Times New Roman',
+    url: '',
```

Comment:
We've been loading the font from a CDN in `widget/embedded/src/globalStyles.ts` before. Are you sure it's not need anymore?